pom.xml

```
@@ -115,7 +115,7 @@
 
     <scm>
         <url>https://github.com/jenkinsci/run-condition-plugin</url>
-        <connection>scm:git:git://github.com/jenkinsci/run-condition-plugin.git</connection>
+        <connection>git@github.com:jenkinsci/run-condition-plugin.git</connection>
```

Format of the change is incorrect.  It needs to use the correct format and needs to use https for access rather than requiring a private key and ssh.
```suggestion
        <connection>scm:git:https://github.com/jenkinsci/run-condition-plugin.git</connection>
```